app/main.py

```diff
@@ -1,2 +1,52 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity_table = 8
+        self.threshold_table = int(self.capacity_table * (2 / 3))
+        self.size_table = 0
+        self.hash_table = [[] for i in range(self.capacity_table)]
+
+    def __len__(self) -> int:
+        return self.size_table
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        if self.size_table == self.threshold_table:
+            self.resize_table()
+        hash_key = hash(key)
```

Comment:
I think it's better to rename this variable into `key_hash` or something like that, since it would be a more natural way to describe it in English.